android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -206,16 +206,18 @@ public void fetchNativeRelease(Promise promise) {
     public void fetchNativeAppStart(Promise promise) {
         final AppStartState appStartInstance = AppStartState.getInstance();
         final Date appStartTime = appStartInstance.getAppStartTime();
+        final Boolean coldStart = appStartInstance.isColdStart();
 
-        if (appStartTime == null) {
+        if (appStartTime == null || coldStart == null) {
+            logger.warning("App start won't be sent due to missing appStartTime or coldStart.");
             promise.resolve(null);
```

Comment:
`l`: Maybe it makes sent to split this up and also to rename `coldStart` to `isColdStart`
```suggestion
        if (appStartTime == null) {
            logger.warning("App start won't be sent due to missing appStartTime.");
            promise.resolve(null);
        } else if (isColdStart == null) {
            logger.warning("App start won't be sent due to missing isColdStart.");
            promise.resolve(null);
```